### PR TITLE
add likes count

### PR DESCRIPTION
**Pull request by RodPaDev**
# Description

This adds likes to count
**Note: this solution is not optimal, should be resolved with a better SQL query**

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## Change status

- [x] Complete, tested, ready to review and merge

# How Has This Been Tested?

Please describe steps taken to ensure this feature is functional and does not break other features:
Tested on postman
# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have run the app using my feature and ensured that no functionality is broken
- [x] My changes generate no new warnings
- [x] There are no merge conflicts

### CODE REVIEW

File: rest-api/likes/models/likes-models.js [hunk 1]
```diff
@@ -26,9 +35,21 @@ async function getLikedRecipesByUserId(id) {
     .leftJoin('recipe_images', 'recipe_images.recipe_id', 'recipes.id')
     .leftJoin('images', 'images.id', 'recipe_images.image_id')
     .where('likes.user_id', id);
+    
+    const reicpeWithLikes = [];
```
Comment (CarnunMP) [inline rest-api/likes/models/likes-models.js]: Typo! :))

File: rest-api/likes/models/likes-models.js [hunk 2]
```diff
@@ -26,9 +35,21 @@ async function getLikedRecipesByUserId(id) {
...
+
+    // .map() doesn't work with async
+    // need to find SQL query in order to optimize this
+    for(recipe of recipes){
+      reicpeWithLikes.push({
+        ...recipe,
+        ...await getLikesByRecipeID(recipe.id)
```
Comment (CarnunMP) [inline rest-api/likes/models/likes-models.js]: Also, "likes" should just be a number! (Like the "likes" property served by GET all recipes.) 
image

Comment (RodPaDev) [reply rest-api/likes/models/likes-models.js]: COUNT on potgres returns a stringified Number

Comment (RodPaDev) [reply rest-api/likes/models/likes-models.js]: Done fixed.

**Pull request merged**